src/python/py_rag.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <pybind11/functional.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl/filesystem.h>
+#include <pybind11/stl_bind.h>
+
+#include "openvino/genai/rag/text_embedding_pipeline.hpp"
+#include "py_utils.hpp"
+#include "tokenizers_path.hpp"
+
+namespace py = pybind11;
+using ov::genai::EmbeddingResult;
+using ov::genai::TextEmbeddingPipeline;
+
+namespace pyutils = ov::genai::pybind::utils;
+
+void init_rag_pipelines(py::module_& m) {
+    m.doc() = "Pybind11 binding for RAG Pipelines";
```

Comment:
looks like here we override global documentation title for `src/python/openvino_genai/py_openvino_genai.pyi`

I think we need to drop `m.doc()` from all `init_XXX` functions and keep only this description:
https://github.com/openvinotoolkit/openvino.genai/blob/782109739f8a92fbad82819a6584912ace6f92ea/src/python/py_openvino_genai.cpp#L73

Reply:
Thanks, fixed